.alexignore

```diff
@@ -1 +1 @@
-generated/plugins/*
+src/content/plugins/_*.md
```

Comment:
Shouldn't we ignore more paths, like _loaders_? Or all the fetched files go to this one?

Reply:
Yeah